_overview/overview-of-spinnaker.md

```diff
@@ -4,39 +4,63 @@ title: Overview of Armory Spinnaker
 order: 10
 permalink: '/'
 ---
-
-## What is Spinnaker?
+## What Is Spinnaker?
 {:.no_toc}
 * This is a placeholder for an unordered list that will be replaced with ToC. To exclude a header, add {:.no_toc} after it.
 {:toc}
 
-<a href="https://kb.armory.io/spinnaker/what-is-spinnaker"><img height="20" style="float: left;" src="https://drod.io/1h3I273p002U/video-file.png"></a>&nbsp; [Watch Ethan explain Spinnaker in three minutes](https://kb.armory.io/spinnaker/what-is-spinnaker/)
+<a href="https://kb.armory.io/spinnaker/what-is-spinnaker"><img height="20" style="float: left;" src="https://drod.io/1h3I273p002U/video-file.png"></a>[ Watch Ethan explain Spinnaker in three minutes](https://kb.armory.io/spinnaker/what-is-spinnaker/)
+
+Spinnaker is an open source, multi-cloud Continuous Delivery and Deployment platform that provides a single pane of glass with visibility across your deployment for deployment status, infrastructure, security and compliance, and metrics. It uses pipelines, flexible and customizable series of deployment stages, to fit a variety of cloud deployment needs.
+
+Spinnaker can deploy to and manage clusters across Amazon Web Services (AWS), Kubernetes, and Google Cloud Platform (GCP) with full feature compatibility for those cloud providers. Additionally, Spinnaker can deploy to Cloud Foundry, and full integration with Microsoft Azure is currently underway.
```

Comment:
there is not full features for everything at this point. for example i THINK lambda is still in the works